### ISSUE TITLE
Call either gtmOnSucces or gtmOnFailure exactly once every time your template is run

### ISSUE THREAD

**Issue by jonathanleekay2** [unceasing-dev/aaanow-gtm-template#1]
Thank you for your gallery submission. When reviewing your template, we've noticed that there is a case where `gtmOnSuccess` or `gtmOnFailure` are never called if injectScript fails [0].  Consider calling gtmOnFailure for this case [1]

* [0] https://github.com/unceasing-dev/aaanow-gtm-template/blob/da2ecafcd2194edf401d2798b2e0f7b3c0c8be2e/template.tpl#L71
* [1] https://developers.google.com/tag-platform/tag-manager/templates/api#injectscript

**Comment by jribbens** [comment]
@jonathanleekay2 Thanks. I have pushed an update which fixes this. Do I need to do anything else to notify Google of the update?

I must admit I have found the documentation for the "sandboxed javascript" and its environment to be extremely hard to follow. I would certainly have added the call to `gtmOnFailure` originally if I had known that was what I was supposed to do!

**Comment by jonathanleekay2** [comment]
Thanks for the quick response. Since you also updated the version in metadata.yaml, we will be able to continue the review. 

In the meantime, we'll take a look at the documentation and see what updates would make it more clear.
